clicksend_sms_client/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,4 @@
+To configure this module, you need to:
+
+* go to settings > technical > telephony > gateway configuration and select ClickSend http in method
```

Comment:
If the users are going to use a specific method of a specific api, wouldn't it be easier for them to have the `self.gateway_id.url` by default set to the api's url? And then they will have the opportunity to change it of course. Also, I think it would save them some trouble down the road if, on the click of `Save` on the create/write of a `sms.gateway` record, for a verification to be shown to the user of whether the data they provided are correct? By invoking the `Account Balance` functionality for example.